### ISSUE TITLE
noise recording on 10.10.5 

### ISSUE THREAD

**Issue by dttlgotv** [ExistentialAudio/BlackHole#222]
I use BlackHole to record system audio on  10.10.5 mac.  
Some noise can be  heard. By the way  on 10.15.4 no nose can be produced.

**Comment by devinroth** [comment]
Good to know. I’ll add it to the todo list.

**Comment by ecogedi** [comment]
Is there any progress on this issue? I have what appears to be a similar problem that I posted as #223  
The problem persists. Thank you.

**Comment by dttlgotv** [comment]
> 
> 
> Is there any progress on this issue? I have what appears to be a similar problem that I posted as #223
> The problem persists. Thank you.

no idea about this issue. I just wait for @devinroth

**Comment by devinroth** [comment]
Could you send me some screen shots?

Devin

**Comment by ecogedi** [comment]
Hi,
I included screen shots as a pdf file in the original comment I posted
(#223) on gituhub, but here they are again. Im getting a new MOTU M4
delivered tomorrow, and will let you know if that changes the periodic
static vs. the now very old MOTU 828. I somehow dont think so.  We
practiced with jamkazam on Sunday and every few minutes you will get about
10-15 secs of static noise, enough to drown out the signal considerably.

Note that in the original screenshot I did not have drift correction
enabled for blackhole, but subsequently did, and that is not the problem (I
have both built-in output and blackhole using drift correcting for the
aggregate device and letting MOTU 828 clock be the master.

Thanks!
